Description: Make Ctrl+Q work on normal X11 desktops
 Upstream changed from explicit Ctrl+Q to QkeySequence::Quit, which Qt defines
 to Ctrl+Q for OSX, KDE and Gnome, but not for other X11 desktops, so
 explicitely add Ctrl+Q back.
Author: Jakob Haufe <sur5r@sur5r.net>
Forwarded: no
Last-Update: 2012-02-16

--- minitube-1.7.orig/src/MainWindow.cpp
+++ minitube-1.7/src/MainWindow.cpp
@@ -281,7 +281,9 @@ void MainWindow::createActions() {
 
     quitAct = new QAction(tr("&Quit"), this);
     quitAct->setMenuRole(QAction::QuitRole);
-    quitAct->setShortcut(QKeySequence(QKeySequence::Quit));
+    quitAct->setShortcuts(QList<QKeySequence>()
+                          << QKeySequence(QKeySequence::Quit)
+                          << QKeySequence(Qt::CTRL + Qt::Key_Q));
     quitAct->setStatusTip(tr("Bye"));
     actions->insert("quit", quitAct);
     connect(quitAct, SIGNAL(triggered()), SLOT(quit()));
